x-pack/plugins/session_view/server/routes/alert_status_route.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+import type { ElasticsearchClient } from 'kibana/server';
+import { IRouter } from '../../../../../src/core/server';
+import { ALERT_STATUS_ROUTE, ALERTS_INDEX, ALERT_UUID_PROPERTY } from '../../common/constants';
+import { expandDottedObject } from '../../common/utils/expand_dotted_object';
+
+export const registerAlertStatusRoute = (router: IRouter) => {
+  router.get(
+    {
+      path: ALERT_STATUS_ROUTE,
+      validate: {
+        query: schema.object({
+          alertUuid: schema.string(),
+        }),
+      },
+    },
+    async (context, request, response) => {
+      const client = context.core.elasticsearch.client.asCurrentUser;
+      const { alertUuid } = request.query;
+      const body = await searchAlertByUuid(client, alertUuid);
+
+      return response.ok({ body });
+    }
+  );
+};
+
+export const searchAlertByUuid = async (client: ElasticsearchClient, alertUuid: string) => {
+  const search = await client.search({
```

Comment:
We may want to update this to use the AlertsClient once I've merged my alerts tab work. It handles authorized indexes and other rbac type stuff.

Reply:
Sounds good, I can do it in a separate PR since the flyout callback is also needed in your PR. I will get this one merged in first